x/accesscontrol/keeper/keeper_test.go

```diff
@@ -122,6 +122,92 @@ func TestWasmDependencyMapping(t *testing.T) {
 	require.Error(t, aclkeeper.ErrWasmDependencyMappingNotFound, err)
 }
 
+func TestWasmDependencyMappingWithExecuteMsgInfo(t *testing.T) {
+	app := simapp.Setup(false)
+	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
+
+	wasmContractAddresses := simapp.AddTestAddrsIncremental(app, ctx, 1, sdk.NewInt(30000000))
+	wasmContractAddress := wasmContractAddresses[0]
+	wasmMapping := acltypes.WasmDependencyMapping{
```

Comment:
One thing I realized for this new wasmdependencyMapping is that we need to update the validation logic in the message_dependency_mapping.go file. More specifically, do we want  to enforce COMMIT access op ONLY for the base access ops?

Reply:
yeah that's what we currently do